LICENSE

```diff
@@ -1,63 +1,26 @@
-This library (libraries/base) is derived from code from two
-sources: 
-
-  * Code from the GHC project which is largely (c) The University of
-    Glasgow, and distributable under a BSD-style license (see below),
-
-  * Code from the Haskell 98 Report which is (c) Simon Peyton Jones
-    and freely redistributable (but see the full license for
-    restrictions).
-
-The full text of these licenses is reproduced below.  Both of the
-licenses are BSD-style or compatible.
-
------------------------------------------------------------------------------
-
-The Glasgow Haskell Compiler License
-
-Copyright 2004, The University Court of the University of Glasgow. 
+Copyright (c) 2017, Carter Tazio Schonwald
```

Comment:
This is a package created by University of Glasgow. We can't just change the license, it must stay the same!